tests/conftest.py

```diff
@@ -159,3 +161,61 @@ def multihot_dataset(multihot_data):
     ds = Dataset(make_df(multihot_data))
     ds.schema["Post"] = ds.schema["Post"].with_tags(Tags.TARGET)
     return ds
+
+
+@pytest.fixture(scope="session")
+def num_embedding_ids():
+    return 1
+
+
+@pytest.fixture(scope="session")
+def embeddings_part_size():
+    return 1e5
+
+
+@pytest.fixture(scope="session")
+def embedding_ids(num_embedding_ids, embeddings_part_size):
+    df = make_df({"id": np.arange(num_embedding_ids * embeddings_part_size).astype("int32")})
+    return df
+
+
+@pytest.fixture(scope="session")
+def rev_embedding_ids(embedding_ids, tmpdir_factory):
```

Comment:
reverse embeddings is used to ensure that id_lookup is working correctly, in this case the indexes are reversed, [99999:1] , In embedding_ids above its [1:99999]. This allows us to use enumeration of batches to pull out the correct (what should be in the embeddings) values and assert they are what came back in each batch.